qa/acceptance-tests/src/test/java/io/camunda/it/util/TestHelper.java

```diff
@@ -244,6 +244,33 @@ public static void waitForScopedProcessInstancesToStart(
             });
   }
 
+  /**
+   * Waits for active process instances which are having a process variable {@link
+   * #VAR_TEST_SCOPE_ID}.
+   *
+   * @param camundaClient ... CamundaClient
+   * @param scopeId ... some unique id for the test case
+   */
+  public static void waitForScopedActiveProcessInstances(
+      final CamundaClient camundaClient, final String scopeId, final int expectedProcessInstances) {
+    Awaitility.await("should start process instances and import in Operate")
```

Comment:
🔧 We don't wait for Operate import, we don't import anymore with the new exporter structure 🙂 Can we generally move to something like "should wait until XYZ is/are available"?

```suggestion
    Awaitility.await("should wait until process instances are available")
```